samples/cellular/modem_shell/README.rst

```diff
@@ -819,6 +819,26 @@ You can use the ``heap`` command to print kernel and system heap usage statistic
      free:              160
      allocated:          88
 
+----
+
+GPIO pin pulse counter
+======================
+
+You can use the ``gpio_count`` command to count pulses on a given GPIO pin.
+When pulse counting has been enabled, **LED 2** on nRF91 Series DKs shows the state of the pin input.
+
+.. note::
+
+   The ``gpio_count enable`` command configures the GPIO pin as input and enables pull down.
+
+.. code-block:: console
+
+   mosh:~$ gpio_count get
+   Number of pulses: 0
+   mosh:~$ gpio_count enable 10
+   mosh:~$ gpio_count get
+   Number of pulses: 42
+
```

Comment:
Maybe add `gpio_count disable` command and get after that if it returns a value (I think it does based on code).